documentation/course/E1.md

```diff
@@ -708,37 +708,180 @@ classDiagram
+1. `TIC/06` - Jako _pasażer_ chcę mieć możliwość wyświetlenia listy swoich zakupionych _biletów_.
+2. `TIC/11` - Jako _pasażer_ chcę mieć możliwość zobaczenia historii zakupionych _biletów_.
 
 #### Mapa nawigacyjna
+```mermaid
+classDiagram
+    direction LR
+
+    class EkranZakupuBiletu {
+        <<ServerPage>>
+        }
+
+    class PasekNawigacyjny {
+        <<ServerPage>>
+        }
+
+    class EkranTwoichBiletow {
+        <<ServerPage>>
+        listaAktywnychBiletow
+        listaArchiwalnychBiletow
+        waznosciBiletow
+        cenyBiletow
+    }
+
+    class EkranSzczegolowBiletu {
+        <<ServerPage>>
+        }
+        
+    class EkranOfertyBiletow {
+        <<ServerPage>>
+        }
+
+    PasekNawigacyjny ..> EkranTwoichBiletow: «Navigates»\n«StartUseCase»\n{ event = btnBilety }
+    PasekNawigacyjny ..> EkranOfertyBiletow: «Navigates»\n{ event = btnKupBilet }
+    EkranTwoichBiletow ..> PasekNawigacyjny: «Navigates»\n«EndUseCase»\n{ event = dowolnyPrzycisk }
+
+    EkranTwoichBiletow ..> EkranSzczegolowBiletu: «Navigates»\n«EndUseCase»\n{ event = btnPokazDoKontroli }
+    EkranSzczegolowBiletu ..> EkranTwoichBiletow: «Navigates»\n«StartUseCase»\n{ event = btnPowrot }
+
+    EkranTwoichBiletow ..> EkranZakupuBiletu: «Navigates»\n«EndUseCase»\n{ event = btnKupPonownie }
+    EkranZakupuBiletu ..> EkranTwoichBiletow: «Navigates»\n«StartUseCase»\n{ event = btnPowrot }
+
+    EkranTwoichBiletow ..> EkranOfertyBiletow: «Navigates»\n«EndUseCase»\n{ event = btnSklep }
+    EkranOfertyBiletow ..> EkranTwoichBiletow: «Navigates»\n«StartUseCase»\n{ event = btnTwojeBilety }
+   
+    EkranOfertyBiletow ..> EkranZakupuBiletu: «Navigates»\n{ event = btnKup }
+    EkranZakupuBiletu ..> EkranOfertyBiletow: «Navigates»\n{ event = btnPowrot } 
+    
+```
 
 #### Prototypy ekranów
 ![TIC/06](images/Ticket1.png)
 
 
-### `TIC/07` - Jako _pasażer_ chcę mieć możliwość wyświetlenia szczegółów zakupionego _biletu_.
+### Wymaganie 6 (Jakub Zehner) 
+1. `TIC/10` - Jako _pasażer_ chcę mieć możliwość zobaczenia _oferty biletów_.
 
 #### Mapa nawigacyjna
+```mermaid
+classDiagram
+    direction LR
+
+    class EkranOfertyBiletow {
+        <<ServerPage>>
+        listaBiletowJednorazowych
+        listaBiletowCzasowych
+        listaBiletowOkresowych
+        }
+
+    class PasekNawigacyjny {
+        <<ServerPage>>
+        }
+
+    class EkranTwoichBiletow {
+        <<ServerPage>>
+    }
+
+    class EkranZakupuBiletu {
+        <<ServerPage>>
+        }
+
+    PasekNawigacyjny ..> EkranTwoichBiletow: «Reloads»\n{ event = btnBilety }
+    PasekNawigacyjny ..> EkranOfertyBiletow: «Reloads»\n«StartUseCase»\n{ event = btnKupBilet }
+    EkranOfertyBiletow ..> PasekNawigacyjny: «Navigates»\n«EndUseCase»\n{ event = dowolnyPrzycisk }
+
+    EkranOfertyBiletow ..> EkranZakupuBiletu: «Navigates»\n«EndUseCase»\n{ event = btnKup }
+    EkranZakupuBiletu ..> EkranOfertyBiletow: «Navigates»\n«StartUseCase»\n{ event = btnPowrot }
```

Comment:
Is this good that the same button navigates to diffrent pages? (btnPowrot)

Reply:
I don't know how it would be any different, `Powrot` goes back to the previous screen, so it depends from which one you entered. Having another button would just make the frontend implementation more difficult.